docs/readmes/orc8r/dev_sub_digests.md

```diff
@@ -0,0 +1,88 @@
+---
+id: dev_sub_digests
+title: Subscriber Digests
+hide_title: true
+---
+
+# Subscriber Digests
+
+Orc8r manages subscriber data through its `subscriberdb` service, and streams the data southbound to AGWs on a regular basis. In this process, a digest-cache pattern is used to intelligently recognize and send down only the data necessary for syncing AGWs. This document overviews the said pattern.
+
+## Motivation
+
+Previously, orc8r follows a vanilla data transmission pattern that sends all subscriber data down to all gateways at every configurable interval.
+
+This introduces various issues at scale, a major one being the significant amount of network pressure this pattern necessitates. For example, to achieve our [GA scalability targets](https://docs.google.com/document/d/1P306DmuC1CFi7bqz4_VVi7v4Ewr1y-dD47C-s5XzdJ8/edit), we need to support **~20TB of network load per month per network**, which is definitively unsustainable.
+
+Therefore, starting from v1.6, as part of our [subscriber scaling project](../proposals/p010_subscriber_scaling.md), orc8r implements the subscriber digests pattern to minimize network pressure and optimize the current mode of subscriber data transmission. This pattern utilizes the assumption of reads >> writes, and focuses on two major improvements
+1. Instead of syncing at every interval, only sync when AGW is out-of-sync
+2. Instead of syncing all data, sync the cloud-AGW data changeset when reasonable
+
+## Overview
+
+The architectural additions implemented for the subscriber digests pattern are highlighted below.
+
+![Subscriber Digests Architecture](assets/orc8r/subscriber_digests_architecture.png)
+
+### Digest Generation
+
+This pattern relies on taking deterministically encoded data snapshots, a.k.a. "digests", that are used to represent the local data version. To this end, orc8r library code `mproto` is created using the golang third-party library `proto` to conduct consistently deterministic data serializations.
+
+The subscriber digests pattern specifically generates and utilizes two types of digests
+
+1. **Flat digests**: a digest of the entire set of subscriber data of a network
```

Comment:
As an aside: after we iterate on the digest interfaces to make them more generic, be sure to come back through this doc and lightly update the verbiage